tests/conftest.py

```diff
@@ -34,6 +38,9 @@
         ColorInterp.alpha,
     ]
 
+s3_client = boto3.client("s3", endpoint_url=AWS_ENDPOINT_URI)
+s3_client.upload_file(TEST_FILE, "gfw-data-lake-test", "wur_radd_alerts/v20201214/raster/epsg-3857/zoom_14/rgb_encoded/geotiff/000R_000C.tif")
```

Comment:
I noticed I hard-coded the data lake bucket name in the lambda function. Best would be to import the name from core state and expose it as ENV Var in the lambda function. For the tests you would set the var in docker compose.